lms/validation/_api.py

```diff
@@ -30,3 +31,25 @@ class APIRecordSubmissionSchema(PyramidRequestSchema):
     Opaque identifier provided by the LMS to identify a submission. This
     typically encodes the assignment context and LMS user.
     """
+
+
+class APIRecordResultSchema(PyramidRequestSchema):
+    """Schema for validating proxy requests to LTI Outcomes API for recording grades."""
+
+    locations = ["json"]
+
+    lis_outcome_service_url = fields.Str(required=True)
+    """URL provided by the LMS to submit grades or other results to."""
+
+    lis_result_sourcedid = fields.Str(required=True)
+    """
+    Opaque identifier provided by the LMS to identify a submission. This
+    typically encodes the assignment context and LMS user.
+    """
+
+    score = fields.Number(
```

Comment:
This seems to work quite nicely but could use an extra set of eyes to double-check.

Reply:
I think it really is as simple as it looks. Your unit tests look sufficient to me, so that proves it